src/tasks/minions/christmasActivity.ts

```diff
@@ -0,0 +1,55 @@
+import { roll } from 'e';
+import { Task } from 'klasa';
+import { LootTable } from 'oldschooljs';
+
+import { antiSantaOutfit } from '../../lib/data/CollectionsExport';
+import { ChristmasTaskOptions } from '../../lib/types/minions';
+import { handleTripFinish } from '../../lib/util/handleTripFinish';
+import resolveItems from '../../lib/util/resolveItems';
+
+const foodTable = new LootTable()
+	.add('Bucket of milk')
+	.add('Carrot')
+	.add('Chocolate kebbit')
+	.add('Chocolate bar')
+	.add('Strawberry');
+
+const stealItems = resolveItems(['Yo-yo']);
+const deliverItems = resolveItems(['Snow globe', 'Candy cane']);
+
+export default class extends Task {
+	async run(data: ChristmasTaskOptions) {
+		const { channelID, quantity, userID, action } = data;
+		const user = await this.client.fetchUser(userID);
+
+		const loot = foodTable.roll();
+		const table = action === 'steal' ? stealItems : deliverItems;
+
+		const cl = user.cl();
+		for (const item of table) {
+			if (cl.has(item)) break;
+			loot.add(item);
+		}
+
+		if (!cl.has('Antisanta mask') && action === 'steal') {
+			loot.add(antiSantaOutfit);
+		}
+
+		if (roll(120)) {
```

Comment:
This rate might want to be looked at. 

Rule of thumb is, whatever the chance is, that many attempts is only a 63% chance to get the item.

Here, 200 attempts (100 hours) @ 1/120 is only an 82% chance to get 1 item.

Obviously some people will be much luckier, and the distribution overall will be 1 per 120, but thats because 'lucky' players get multiple,  while unlucky players are **very** unlucky. I would make this just a **little** more generous, considering how close X-mass is  and how busy people are.